pages/index.vue

```diff
@@ -4,8 +4,9 @@ div
     .container
       h1.display-4 Benieuwd naar het digitale aanbod van Yep?
       p.lead
-        | Doordat fysieke trainingen in deze tijd niet altijd wenselijk zijn,
+        | Doordat fysieke trainingen in deze coronatijd niet altijd wenselijk zijn,
```

Comment:
goed idee, maar zou het zo doen:
```suggestion
        | Doordat fysieke trainingen in deze corona tijd niet altijd wenselijk zijn,
```